pkg/scheduler/plugins/nodeorder/nodeorder.go

```diff
@@ -368,13 +379,17 @@ func taintTolerationScore(
 	}
 
 	nodescoreList := make(k8sframework.NodeScoreList, len(nodes))
-	errCh := make(chan error, len(nodes))
-	workqueue.ParallelizeUntil(context.TODO(), 16, len(nodes), func(index int) {
+	// size of errCh should be ge than parallelization number, see interPodAffinityScore.
```

Comment:
what is `ge`?

Reply:
"greater or equal"
I can make it more clear

Reply:
That's the better. I think there is no need to abbreviate here.

Reply:
Got it!